### PR TITLE
Address PR comments

### PR DESCRIPTION
**Pull request by JayaShakthi97**
### Related Issues
- N/A

### Related PRs
- https://github.com/wso2/identity-apps/pull/2917
- https://github.com/wso2/identity-apps/pull/2922
- https://github.com/wso2/identity-apps/pull/2951
- https://github.com/wso2/identity-apps/pull/2959
- https://github.com/wso2/identity-apps/pull/3090
- https://github.com/wso2/identity-apps/pull/3289

### Checklist
- [ ] e2e cypress tests locally verified.
- [ ] Manual test round performed and verified.
- [ ] UX/UI review done on the final implementation.
- [ ] Documentation provided. (Add links if there are any)
- [ ] Unit tests provided. (Add links if there are any)
- [ ] Integration tests provided. (Add links if there are any)

### Security checks
- [ ] Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines?
- [ ] Ran FindSecurityBugs plugin and verified report?
- [ ] Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets?

### CODE REVIEW

File: apps/console/src/features/applications/components/forms/inbound-oidc-form.tsx [hunk 1]
```diff
@@ -2338,7 +2342,8 @@ export const InboundOIDCForm: FunctionComponent<InboundOIDCFormPropsInterface> =
 
     /**
      * Renders the application secret regenerate confirmation modal.
-     * @return {ReactElement}
+     * 
+     * @returns Regenerate confirmation modal.
```
Comment (pavinduLakshan) [inline apps/console/src/features/applications/components/forms/inbound-oidc-form.tsx:2346]:
```suggestion
     * @returns the modal for confirming regenerating app secret.
```

Comment (JayaShakthi97) [reply apps/console/src/features/applications/components/forms/inbound-oidc-form.tsx]: 8f72cfb7f41f31d2e509d407a6482a7f6ecb630e

**Review by Yasara-Y — APPROVED** [review]

**Review by pavinduLakshan — APPROVED** [review]

File: apps/console/src/features/applications/components/forms/inbound-oidc-form.tsx [hunk 2]
```diff
@@ -1,7 +1,7 @@
 /**
- * Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ * Copyright (c) 2020, WSO2 LLC. (https://www.wso2.com). All Rights Reserved.
```
Comment (mevan-karu) [inline apps/console/src/features/applications/components/forms/inbound-oidc-form.tsx:2]: Update year also.

**Review by mevan-karu — APPROVED** [review]

File: apps/console/src/features/applications/components/forms/inbound-oidc-form.tsx (hunk 2 above)
Comment (JayaShakthi97) [reply apps/console/src/features/applications/components/forms/inbound-oidc-form.tsx:2]: Since this is an old file, the year won't be updated.

**Pull request merged**